config/dev.exs

```diff
@@ -4,7 +4,10 @@ config :amqp,
   connections: [
     roger_conn: [
       host: "localhost",
-      port: 5672
+      port: 5672,
+      username: "user",
```

Comment:
It should not be necessary to add our credentials to a dev-environment.
In test it should be possible, but on dev the credentials should be taken from the including application.